cmake/Modules/FindNCCL.cmake

```diff
@@ -33,6 +33,23 @@
 
 set(NCCL_ROOT_DIR "" CACHE PATH "Folder contains NVIDIA NCCL")
 
+# first check in the /usr/local/cuda before other paths
+if (UNIX)
+  set (search_paths "/usr/local/cuda")
+
+  find_path(NCCL_INCLUDE_DIRS
+  NAMES nccl.h
```

Comment:
Not familiar with cmake, but do you need add indentation here?

Reply:
yes good catch! indentation needed.